pkgs/build-support/vm/default.nix

```diff
@@ -1168,7 +1195,7 @@ rec {
     "passwd"
   ];
 
-  commonDebianPackages = commonDebPackages ++ [ "sysvinit" "diff" "mktemp" ];
+  commonDebianPackages = commonDebPackages ++ [ "sysvinit" "diff" ];
```

Comment:
Can you add a note about the reason for the removal of this in the commit message?

Reply:
done